index.html

```diff
@@ -494,7 +494,7 @@ <h2>The <dfn>WOT</dfn> namespace</dfn></h2>
     <a href="https://www.w3.org/TR/2019/CR-wot-thing-description-20190516/#form">
     Form</a> contains a `contentType` to describe the data.
     For certain content types, a <a>DataSchema</a> is defined, based on
-    <a>JSON schema</a>, making possible to represent these contents as
+    <a>JSON Schema</a>, making possible to represent these contents as
```

Comment:
If we do this change HERE we need to do the same change everywhere else!

Reply:
Please do so :)

Reply:
I did :-)